core/src/main/scala/opscenter-frees.scala

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
No need to have `-frees` in the file name

Reply:
Changed in the latest commit